node/src/main/scala/coop/rchain/node/api/WebApi.scala

```diff
@@ -335,23 +329,53 @@ object WebApi {
   private def exprFromBundleProto(b: Bundle): Option[RhoExpr] = exprFromParProto(b.body)
 
   private def mkUnforgExpr(f: String => RhoUnforg, bs: ByteString): ExprUnforg =
-    ExprUnforg(f(toHex(bs)))
+    ExprUnforg(f(bs.toHexString))
 
   // RhoExpr to protobuf
 
+  private def rhoExprToParProto(exp: RhoExpr): Par = {
+    def exprProtoToParProto(exp: Expr): Par = Par(exprs = Seq(exp))
```

Comment:
I always forgot that we have nice constructors for proto in RhoType.
https://github.com/rchain/rchain/blob/0bfc2aa13e73a27c1f21200dee7336bd252ea17a/rholang/src/main/scala/coop/rchain/rholang/interpreter/RhoType.scala#L130-L134